storage/persistent_storage/persistent-storage-ocs.adoc

```diff
@@ -15,22 +15,40 @@ Red Hat OpenShift Container Storage provides its own documentation library. The
 
-|Deploying Red Hat OpenShift Container Storage 4.4 on an existing {product-title} cluster
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.4/html/deploying_openshift_container_storage/index[Deploying Red Hat OpenShift Container Storage 4.4]
+|Instructions on preparing to deploy when your environment is not directly connected to the internet
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/preparing_to_deploy_in_a_disconnected_environment/index[Preparing to deploy OpenShift Container Storage 4.5 in a disconnected environment]
 
-|Managing a Red Hat OpenShift Container Storage 4.4 cluster
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.4/html/managing_openshift_container_storage/index[Managing Red Hat OpenShift Container Storage 4.4]
+|Instructions on deploying OpenShift Container Storage using Amazon Web Services for local or cloud storage
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_openshift_container_storage_using_amazon_web_services/index[Deploying OpenShift Container Storage 4.5 using Amazon Web Services]
 
-|Monitoring a Red Hat OpenShift Container Storage 4.4 cluster
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.4/html/monitoring_openshift_container_storage/index[Monitoring Red Hat OpenShift Container Storage 4.4]
+|Instructions on deploying OpenShift Container Storage to local storage on bare metal infrastructure
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_openshift_container_storage_using_bare_metal_infrastructure/index[Deploying OpenShift Container Storage 4.5 using bare metal infrastructure]
 
-|How to troubleshoot errors and issues in OpenShift Container Storage
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.4/html/troubleshooting_openshift_container_storage/[Troubleshooting OpenShift Container Storage]
+|Instructions on deploying OpenShift Container Storage on Red Hat OpenShift Container Platform VMWare vSphere clusters
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_openshift_container_storage_using_vmware/index[Deploying OpenShift Container Storage 4.5 using VMWare]
+
+|Instructions on deploying OpenShift Container Storage to use an external Red Hat Ceph Storage cluster
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_openshift_container_storage_with_external_storage/index[Deploying OpenShift Container Storage 4.5 with external storage]
+
+|Instructions on deploying and managing OpenShift Container Storage on existing Red Hat OpenShift Container Platform Google Cloud clusters
```

Comment:
Instead of "Red Hat OpenShift Container Platform", you can use the variable `{product-title}`. However, I'm not sure that "OpenShift Container Platform Google Cloud Platform clusters" is the right description. 

IIUC, you should be able to drop OCP and just say "...on existing Google Cloud Platform clusters."

Alternatively, you could match with your AWS wording and say "...deploying OpenShift Container Storage using Google Cloud Platform for local or cloud storage"

Either way, it would be good to have consistent usage throughout all the rows in this table.